satellite/gracefulexit/getnodes_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package gracefulexit_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/internal/testcontext"
+	"storj.io/storj/internal/testrand"
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/satellite"
+	"storj.io/storj/satellite/overlay"
+	"storj.io/storj/satellite/satellitedb/satellitedbtest"
+)
+
+func TestSatelliteDBSetup(t *testing.T) {
+	satellitedbtest.Run(t, func(t *testing.T, db satellite.DB) {
+		ctx := testcontext.New(t)
+		defer ctx.Cleanup()
+
+		testGetExitingNodes(ctx, t, db.OverlayCache())
+	})
+}
+
+func testGetExitingNodes(ctx context.Context, t *testing.T, cache overlay.DB) {
+	for _, tt := range []struct {
+		numNodesToExit       int
+		nodesTotal           int
+		expectedExitingNodes int
+	}{
+		{2, 2, 2},
```

Comment:
Not sure if this is necessary. Maybe we should set the `nodesTotal` to be larger than `numNodesToExit` so we can check that we are only getting the nodes that are existing?

Reply:
yeah this is the first case to make sure the test set up is working at all

Reply:
Test updated